packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -345,7 +355,23 @@ export class ProjectActions {
     this.api.insertProjectPreferencesToCache(this.ctx.lifecycleManager.projectTitle, args)
   }
 
-  async codeGenSpec (codeGenCandidate: string, codeGenType: CodeGenType): Promise<NexusGenUnions['GeneratedSpecResult']> {
+  async getReactComponentsFromFile (filePath: string): Promise<ReactComponentDescriptor[]> {
+    const project = this.ctx.currentProject
+
+    assert(project, 'Cannot create spec without currentProject.')
+
+    try {
+      const src = await fs.readFile(this.ctx.path.join(project, filePath), 'utf8')
```

Comment:
I think we can use this method: https://github.com/cypress-io/cypress/blob/develop/packages/data-context/src/actions/FileActions.ts#L11

Reply:
I actually think that this should be an absolute path to the file, so we won't need to do this anyway